lib/cli/configuration.js

```diff
@@ -325,3 +326,21 @@ exports.load = (file) => {
     throw error;
   }
 };
+
+/**
+ * @summary Returns defualt configuration file path
+ * @function
+ * @public
+ *
+ * @param {array} fallbacks - file paths for files that may or may not exist
+ * @returns {string} file path
+ *
+ * @example
+ * const configPath = configuration.getDefault([ './foo.js', './bar.js' ]);
+ * console.log(configPath);
+ */
+exports.getDefaultPath = (fallbacks) => {
```

Comment:
Yea but calling it'd you'd always do smthing like `config.getDefualtPath()` so it'd be implied. Anyway after sleeping on it I thought you were right and `firstExistingFile` was better :)

Thanks for the help!